File: AN_2/SEM_2/MPP/java/lab4/src/main/java/ro/mpp/labfx/domain/CursaDTO.java
package ro.mpp.labfx.domain;

import javafx.beans.property.IntegerProperty;
import javafx.beans.property.SimpleIntegerProperty;
import javafx.beans.property.SimpleStringProperty;
import javafx.beans.property.StringProperty;

public class CursaDTO {
    private final StringProperty nume;
    private final IntegerProperty capacitate_min;
    private final IntegerProperty capacitate_max;
    private final IntegerProperty nrParticipanti;

    public CursaDTO(String nume, int capacitate_min, int capacitate_max ,int nrParticipanti) {
        this.nume = new SimpleStringProperty(nume);
        this.capacitate_min = new SimpleIntegerProperty(capacitate_min);
        this.capacitate_max = new SimpleIntegerProperty(capacitate_max);
        this.nrParticipanti = new SimpleIntegerProperty(nrParticipanti);
    }

    public StringProperty getNumeProperty() {
        return nume;
    }

    public IntegerProperty getCapacitateMinProperty() {
        return capacitate_min;
    }

    public IntegerProperty getCapacitateMaxProperty() {
        return capacitate_max;
    }

    public IntegerProperty getNrParticipantiProperty() {
        return nrParticipanti;
    }

    public String getNumeDTO() {
        return nume.get();
    }

    public int getCapacitateMinDTO() {
        return capacitate_min.get();
    }

    public int getCapacitateMaxDTO() {
        return capacitate_max.get();
    }

    public int getNrParticipantiDTO() {
        return nrParticipanti.get();
    }
}

File: AN_2/SEM_2/MPP/c#/lab5/Model/mpp.model/CursaDTO.cs
using MessagePack;
using System.ComponentModel;

namespace Model.mpp.model
{
    [Serializable]
    public class CursaDTO
    {
        public string Nume { get; set; }
        public int CapacitateMin { get; set; }
        public int CapacitateMax { get; set; }
        public int NrParticipanti { get; set; }

        public CursaDTO() {}

        public CursaDTO(string nume, int capacitateMin, int capacitateMax, int nrParticipanti)
        {
            Nume = nume;
            CapacitateMin = capacitateMin;
            CapacitateMax = capacitateMax;
            NrParticipanti = nrParticipanti;
        }
    }
}